packages/playground/backend-integration/vite.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path')
+const glob = require('fast-glob')
+
+/**
+ * @returns {import('vite').Plugin}
+ */
+function BackendIntegrationExample() {
+  return {
+    name: 'backend-integration',
+    config() {
+      const projectRoot = __dirname
+      const sourceCodeDir = path.join(projectRoot, 'frontend')
+      const root = path.join(sourceCodeDir, 'entrypoints')
+      const outDir = path.relative(root, path.join(projectRoot, 'dist/dev'))
+
+      const entrypoints = glob
+        .sync(`${root}/**/*`, { onlyFiles: true })
+        .map((filename) => [path.relative(root, filename), filename])
```

Comment:
@patak-js Is it possible that this glob is invalid in Windows?

From the error in the test it looks like no files are matched. What would be the cross-platform compatible way to write it?

Reply:
Get `normalizePath` from Vite, that converts windows '\' to '/'. Tested locally and it works in Windows:
```js
const entrypoints = glob
        .sync(`${normalizePath(root)}/**/*`, { onlyFiles: true })
        .map((filename) => [path.relative(root, filename), filename])
```

Reply:
Makes sense, in Windows I'm mixing backslashes and slashes in this glob which makes it invalid.

Thanks for testing it and sharing a solution!